site/content/en/docs/Third Party Content/podcasts.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Third Party Podcasts"
+linkTitle: "Podcasts"
+date: 2020-01-15
+description: "Community contributed podcasts on Agones."
+weight: 20
+---
+
+- [Google Cloud Platform Podcast - #202 Supersolid with Kami May](https://gcppodcast.com/post/episode-202-supersolid-with-kami-may/)
+- [Google Cloud Platform Podcast - #142 Agones with Mark Mandel and Cyril Tovena](https://gcppodcast.com/post/episode-142-agones-with-mark-mandel-and-cyril-tovena/)
```

Comment:
I think that we should keep these in reverse chronological order.

It's really hard to tell what the publish date of the gcp podcast episodes are, but it seems that gcp#142 came before k8s#26 and so this line should be at the bottom of the list.

Reply:
I like it! Should we add the date to the listing as well? I was debating doing that, but wasn't sure.

Reply:
I think that would be a good idea.